styles.css

```diff
@@ -113,33 +114,39 @@ textarea {
 
 input {
   padding: 10px;
+  border-radius: 4px;
 }
 
 textarea {
   height: 100px;
   overflow: scroll;
+  border-radius: 4px;
```

Comment:
Same here with sorting alphabetically.

Reply:
On it, thanks for the spot, I'd rather fix it now